### DISCUSSION TITLE
Filter Data by Keyboard Layout

### DISCUSSION THREAD

**Discussion by dleiferives** [monkeytypegame/monkeytype#1511] [Ideas] [answered]
Just as one is able to sort their data by language or by the type of test, one should be able to sort their data by the keyboard layout they are using.

**Comment by SeerLite** [comment, marked as answer]
You can already do that with tags.You can create tags like "QWERTY", "Colemak-DH", etc. and switch to the right one everytime you switch layout. You can then filter by tag in the account page

**Comment by SvenWebel** [comment]
Tagging works but it is an extra step you can forget. 
If the data which layout was used is saved anyway it would be nice to have easy filters for that.
If that data is not saved by default maybe an option to auto generate tags with the layout would be nice?

**Reply by Miodec** [reply]
Layout data is not saved. I think if you forget to tag a test it falls under user error.

**Reply by SvenWebel** [reply]
Fair enough as we can also edit tags after the fact.
A batch edit option might be nice to have feature. That way it will not spam the server and get you timed out for "Too many requests." (sorry :-))
